modules/@apostrophecms/util/ui/public/http.js

```diff
@@ -118,7 +118,17 @@
     if (options.qs) {
       url = apos.http.addQueryToUrl(url, options.qs);
     }
-
+    if (options.busy) {
```

Comment:
This code was in the "load" event, so we didn't start "looking busy" until the request was already done. Oops. Now we start looking busy when the HTTP request begins, and it looks right.

We really should have a nice spinner and a transition so this fades in and doesn't bug you with flashy-flashy if it only takes an instant, and I don't know why we don't have that in 3.x yet (we do in 2.x), but that's another convo entirely.